DFC.ServiceTaxonomy.GraphSync/GraphSyncers/BagPartGraphSyncer.cs

```diff
@@ -42,8 +42,12 @@ public BagPartGraphSyncer(IContentDefinitionManager contentDefinitionManager, IS
                 var graphSyncer = _serviceProvider.GetRequiredService<IGraphSyncer>();
 
                 string contentType = contentItem!["ContentType"]!.ToString();
+
+                DateTime? createdDate = DateTime.Parse(contentItem["CreatedDate"]!.ToString());
```

Comment:
As `DateTime.Parse` throws if there's a problem with the input string, we should be able to place the result in a non-nullable DateTime and remove the checks for null later on.

Reply:
Updated